src/utill.rs

```diff
@@ -189,6 +193,35 @@ pub fn to_hex(bytes: &Vec<u8>) -> String {
     hex_string
 }
 
+pub fn parse_toml(file_path: &str) -> io::Result<HashMap<String, HashMap<String, String>>> {
```

Comment:
An oneliner doc on this func. Not needed, but later we wanna enforce docs for every function.